GameServerLib/Items/Inventory.cs

```diff
@@ -117,8 +117,9 @@ public void RemoveItem(byte slot, IObjAiBase owner, int stacksToRemove = 1)
         {
             if (stacksToRemove < 0)
             {
-                throw new Exception("Stacks to be Removed can't be a negative number!");
+                stacksToRemove = 0;
             }
+
             var itemID = Items[slot].ItemData.ItemId;
             int finalStacks = Items[slot].StackCount - stacksToRemove;
 
```

Comment:
Remove lines 130 -> 136 (from the original file). This includes the `ItemScripts.Remove()` line.

Reply:
Done